tools/version.txt

```diff
@@ -0,0 +1 @@
+1.15.1-dev
```

Comment:
Reiterating @MicahGale's comment here so we don't miss this

```suggestion
0.15.1-dev
```

Reply:
My bad